src/sass/components/footer.sass

```diff
@@ -112,5 +111,54 @@
       border-width: 6px 6px 0 6px
       border-color: $sub-text transparent transparent transparent
 
+// Language switcher
+.lang-switch-link
+  display: inline-block
+  position: relative
+  transition: all 0s ease
+  svg
+    display: inline-block
+    fill: $sub-text
+    margin-bottom: .15em
+    width: 1em
+    height: 1em
+    vertical-align: middle
+  .footer-country-select__arrow
+    width: .75em
+    height: .75em
+    margin-bottom: .185em
+
+.lang-switch-link:hover
+  .footer-country-select__arrow
+    transform: rotate(180deg)
+  > a
+    color: #fff
+  svg
+    fill: #fff
 
+.lang-popover
+  background: #fff
+  bottom: 40px
+  max-height: 0
+  overflow: hidden
+  right: 0
+  position: absolute
+  width: 150px
+  text-align: left
+  transition-delay: .3s
+  ul
+    border-bottom: 3px solid $brand
+    margin-bottom: 0
+    padding: 10px 20px
+  li
+    list-style: none
+  a
+    color: $text
+    display: block
+    padding: 10px 5px
```

Comment:
We tried to use variables  in spacing as much as possible:

// Spacing
$space-xs: rem(14px)
$space-s: rem(28px)
$space-m: rem(40px)
$space-l: rem(56px)

$space-mob-s: rem(20px)
$space-mob-m: rem(28px)
$space-mob-l: rem(40px)

$space-sec-xs: rem(8px)
$space-sec-s: rem(16px)
$space-sec-m: rem(24px)
$space-sec-l: rem(32px)